client/src/app/component/switch/Switch.stories.tsx

```diff
@@ -0,0 +1,29 @@
+import {ComponentStory, ComponentMeta} from "@storybook/react";
+import {useState} from "react";
+import {Switch as SwitchComponent, SwitchProps} from "./Switch";
+
+export default {
+  title: "Switch",
+  component: SwitchComponent,
+  args: {isLeftPosition: false},
+  argTypes: {isLeftPosition: {type: "boolean"}},
+} as ComponentMeta<typeof SwitchComponent>;
+
+const Template: ComponentStory<typeof SwitchComponent> = (args: SwitchProps) => {
+  const [isOnlineMode, setIsOnlineMode] = useState(false);
```

Comment:
Is it good state for storybook? Let's use something more abstract! Let's talk about it if it's not clear

Reply:
Changed to abstract name